### ISSUE TITLE
plugin type detection on onChange

### ISSUE THREAD

**Issue by gevhambarzumyan93** [vazco/uniforms#967]
I have an issue whid some custom plugin. they must be unique, so I decided to collect a map of the type of the plugin that I use, BUT because I can't know which plugin the user use, I mast loop over the whole data to get theme.

so now I think maybe it won't be hard to make isUnique param which will set hideFromMenu to true after use, or in onChange will be second parameter with used plugin

**Comment by radekmie** [comment]
Hi @gevhambarzumyan93. I have a problem understanding your issue. Could you maybe rephrase it a little? I'm concerned about the word _plugin_ the most - there are no _plugins_ for uniforms, at least I'm not aware of any.
